qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/MuonAnalysis/muon_analysis_2.py

```diff
@@ -482,16 +474,9 @@ def setup_fitting_notifier(self):
         self.fitting_tab.fitting_tab_presenter.update_plot_guess_notifier.add_subscriber(
             self.plot_widget.fit_mode.update_plot_guess_observer)
 
-        #self.model_fitting_tab.model_fitting_tab_presenter.remove_plot_guess_notifier.add_subscriber(
-        #    self.plot_widget.model_fit_mode.remove_plot_guess_observer)
-
-        #self.model_fitting_tab.model_fitting_tab_presenter.update_plot_guess_notifier.add_subscriber(
-        #    self.plot_widget.model_fit_mode.update_plot_guess_observer)
-
     def setup_results_notifier(self):
```

Comment:
Is this functino going to be required in the future? with it now not doing anything is it better to keep it around as a dummy for the future?

Reply:
good spot, we should kill it. I will do that later today